waspc/cli/exe/Main.hs

```diff
@@ -130,6 +133,15 @@ main = withUtf8 . (`E.catch` handleInternalErrors) $ do
       putStrLn $ "\nInternal Wasp error (bug in compiler):\n" ++ indent 2 (show e)
       exitFailure
 
+-- | Sets environment variables that are visible to the commands run by the CLI.
```

Comment:
I would make it clear that they will be set for our CLI, for any processes run by CLI, and will not remain set after CLI finished running (i.e. in the terminal session in which CLI was called, after it).
Righ now it is only clear that it will affect commands run by the CLI.
If you think this is too much info ok, but I know this was not clear to me when we discussed it, to having this info would have been helpful for me.